### DISCUSSION TITLE
Support for Skaffold v2

### DISCUSSION THREAD

**Discussion by dallasvaughan** [GoogleCloudPlatform/cloud-code-intellij#3098] [Q&A] [answered]
Just wondering if plugin support for [Skaffold V2](https://skaffold-v2.web.app/docs/upgrading/) is going to be added? Currently, the plugin will not work if the `skaffold.yaml` is in V2 format (ex:`apiVersion: skaffold/v3alpha1`).

**Comment by etanshaul** [comment, marked as answer]
hey @dallasvaughan - yes, we plan to support Skaffold V2 as soon as we can (no timeline yet, but asap). Could you please share what didn't work for you after upgrading to V2, just to make sure we cover all the bases? Thanks.

**Reply by dallasvaughan** [reply]
Actually I forgot to update the Cloud Code preference to use a custom skaffold version and once I did that it started up and deployed the app successfully.
